CODE_OF_CONDUCT.md

```diff
@@ -1 +1,90 @@
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+FaithTech LABS pledge to making participation in
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
+orientation.
+
+## Our Standards
+
+Examples of behaviour that contributes to creating a positive environment
+include:
+
+* Being friendly and patient
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Trying to understand why we disagree
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behaviour by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+  address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Our Responsibilities
+
+FaithTech is responsible for clarifying the standards of acceptable
+behaviour and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behaviour.
+
+We have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any individual for other behaviours that they deem inappropriate,
+threatening, offensive, or harmful.
+
+## Our Faith
+
+> For it is by grace you have been saved, through faith - and this is not from yourselves, it is the gift of God - not by works, so that no one can boast. For we are God’s handiwork, created in Christ Jesus to do good works, which God prepared in advance for us to do.
+> [Ephesians 2:8-10](https://www.bible.com/bible/111/EPH.2.8-10)
+
+We will always be grounded in our faith and beliefs. Because of our beliefs we will always be open and loving:
+
+> Do everything in love
+> [1 Corinthians 16:14 NIV](https://www.bible.com/bible/111/1co.16.14)
+
+> This is my command: Love each other.
+> [John 15:17 NIV](https://www.bible.com/bible/111/jhn.15.17)
+
+> We love because he first loved us.
+> [1 John 4:19 NIV](https://www.bible.com/bible/111/1jn.4.19)
+
+Faith is a journey and we want to support you wherever you are on that path. We want to help Christians deepen their faith while doing good. We understand if you do not share our beliefs and still welcome you to join us. We will not harass you, but some of our activities may make you feel uncomfortable. You may find our community is not for you and that is okay.
```

Comment:
I suggest tweaking this a bit to make it more relevant to labs than our faith journey in general.

Maybe something like
>Some of our projects are directly connected to doing social good, while others are related to our faith. You are welcome to join us if you don't share our beliefs. We will not harass you or expect you to share our beliefs, but some or our activities and projects may make you...

Reply:
@surajreddy let's talk about this. I see what you are getting at, but am not sure about how to word it.

Reply:
With the recent changes this reads a lot better, but I'm definitely down to chatting!

I just wanted to draw out that some of the projects will be directly related to social good (ex: the app we worked on last year to help track children in Thailand), while some will be very faith related (ex: taking donations for a church).

If this is primarily related to FaithTech Labs projects, I suggest starting with discussing the types of projects and then go into the fact that it's okay for people to be part of our projects even if they don't share our beliefs. But if it's for FaithTech in general, it's fine as is.

Reply:
I am hoping this or something like it can be more for FaithTech in general, but wanted to start with Labs. James has expressed an interest in working on an overall code of conduct.

Reply:
Going with this for now. We can always iterate.